### ISSUE TITLE
`hab pkg build .` and `hab studio new` `-dev` version of `hab-backline`

### ISSUE THREAD

**Issue by rhass** [habitat-sh/habitat#2810]
__OS Version__: `Ubuntu 16.04.2`
__Hab Version__: `0.26.1`

When executing `hab pkg build` or `hab studio new` the `-dev` versions of `hab-backline` are installed by default. The expected behavior is for the latest stable version to be used unless `HAB_DEPOT_CHANNEL` is set to `unstable`.

```
$ hab pkg build .
   hab-studio: Destroying Studio at /hab/studios/home--rhass--src--omnitruck (default)
   hab-studio: Creating Studio at /hab/studios/home--rhass--src--omnitruck (default)
   hab-studio: Importing rhass secret origin key
» Importing origin key from standard input
★ Imported secret origin key rhass-20160729234045.
» Installing core/hab-backline
↓ Downloading core/hab-backline/0.27.0-dev/20170724230749
    2.06 KB / 2.06 KB / [==========================================================] 100.00 % 5.48 MB/s  
```

@fnichol Indicated this is likely fixed in the upcoming 0.27 release via slack: https://habitat-sh.slackarchive.io/general/page-100/ts-1500938458844128

I am opening and assigning this to myself to validate post 0.27 release. If the issue is still present, I will reach out to the team for reassignment/triage.

**Comment by eeyun** [comment]
It's possible that the issue @bodymindarts is seeing here https://github.com/habitat-sh/habitat/issues/2802 might also be related to this. Would probably be worth checking out!

**Comment by rhass** [comment]
@eeyun It looks like this issue is still present in `0.27.0`.

```
$ hab pkg build .
   hab-studio: Destroying Studio at /hab/studios/home--rhass--src--omnitruck (unknown)
   hab-studio: Creating Studio at /hab/studios/home--rhass--src--omnitruck (default)
   hab-studio: Importing rhass secret origin key
» Importing origin key from standard input
★ Imported secret origin key rhass-20160729234045.
» Installing core/hab-backline
↓ Downloading core/hab-backline/0.28.0-dev/20170727204932
    2.06 KB / 2.06 KB / [==========================================================] 100.00 % 6.91 MB/s  
```

**Comment by reset** [comment]
Fixed in 0.28.0, thanks @rhass!
